tests/unit/Application/Metadata/JsonGenerator/ArpGeneratorTest.php

```diff
@@ -136,6 +136,8 @@ private function buildManageAttribute(string $attributeName)
             ->andReturn('idp');
         $attribute->shouldReceive('getValue')
             ->andReturn('The Manage attr value');
+        $attribute->shouldReceive('getMotivation')
```

Comment:
Be aware that this doesn't test anything and if the change in the ARPGenerator is removed this won't result in a failing test.

Reply:
Removing that expectation results in:

```
Mockery\Exception\BadMethodCallException : Received Mockery_1_Surfnet_ServiceProviderDashboard_Infrastructure_Manage_Dto_Attribute::getMotivation(), but no expectations were specified
 /vagrant/src/Surfnet/ServiceProviderDashboard/Application/Metadata/JsonGenerator/ArpGenerator.php:90
 /vagrant/src/Surfnet/ServiceProviderDashboard/Application/Metadata/JsonGenerator/ArpGenerator.php:70
 /vagrant/tests/unit/Application/Metadata/JsonGenerator/ArpGeneratorTest.php:82
```